core/server/master/src/test/java/alluxio/master/AlluxioMasterProcessTest.java

```diff
@@ -72,6 +79,31 @@
   private int mRpcPort;
   private int mWebPort;
 
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
```

Comment:
For this test `waitForServing()` checks if the socket is open. And seems the socket is opened and bounded no matter whether the master is serving requests on them or not. Therefore if you are checking the socket alone, it is not so useful. In order for the test to cover your logic, can you change `waitForServing()` behavior based on whether the server is serving instead of just checking the socket? Try sending an HTTP request or sth?

So far I think updating `waitForServing()` check is the quickest way. @ZhuTopher feel free to suggest on unit test coverage. Thx!

Reply:
Thanks advance.

Reply:
@ZhuTopher Would you please help to give some suggestion? I'm not sure how to add a test to cover this PR changes